docs/tutorial/anonlink-novt.ipynb

```diff
@@ -0,0 +1,942 @@
+   "metadata": {},
+   "source": [
+    "## Solve with Anonlink"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Combining CLKs and Blocks for Party 0\n",
+      "Combining CLKs and Blocks for Party 1\n",
+      "Combining CLKs and Blocks for Party 2\n"
+     ]
+    }
+   ],
+   "source": [
+    "clk_files = ['novt_clk_{}.json'.format(x) for x in range(3)]\n",
+    "block_files = ['novt_blocks_{}.json'.format(x) for x in range(3)]\n",
+    "\n",
+    "clk_blocks = []\n",
+    "\n",
+    "for i, (clk_f, block_f) in enumerate(zip(clk_files, block_files)):\n",
+    "    print('Combining CLKs and Blocks for Party {}'.format(i))\n",
+    "    clk_blocks.append(combine_clks_blocks(open(clk_f, 'rb'), open(block_f, 'rb')))\n",
+    "    \n",
+    "    \n",
+    "clk_groups = []\n",
+    "rec_to_blocks = {}\n",
+    "\n",
+    "for i, clk_blk in enumerate(clk_blocks):\n",
+    "    clk_groups.append(deserialize_filters([r[0] for r in clk_blk]))\n",
+    "    rec_to_blocks[i] = {rind: clk_blk[rind][1:] for rind in range(len(clk_blk))}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assess Precision and Recall"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Example found groups: \n",
+      "((1, 273), (2, 3985))\n",
+      "((1, 2572), (2, 4597))\n",
+      "((1, 2772), (2, 3127), (0, 153))\n",
+      "((1, 3610), (2, 1652), (0, 231))\n",
+      "((0, 2973), (1, 4718), (2, 2278))\n",
+      "((0, 2878), (2, 280))\n",
+      "((0, 4995), (1, 2634), (2, 303))\n",
+      "((0, 3907), (1, 829))\n",
+      "((0, 1993), (1, 2924), (2, 2622))\n",
+      "((1, 6), (2, 8))\n",
+      "\n",
+      "\n",
+      "With blocking: \n",
+      "precision: 0.7808661926308985, recall: 0.7325651910248635\n",
+      "Without blocking: \n",
+      "precision: 0.7808661926308985, recall: 0.7325651910248635\n"
+     ]
+    }
+   ],
+   "source": [
+    "threshold = 0.87\n",
+    "\n",
+    "# matching with blocking\n",
+    "found_groups = solve(clk_groups, rec_to_blocks, threshold)\n",
+    "print(\"Example found groups: \")\n",
+    "for i in range(10):\n",
+    "    print(found_groups[i])\n",
+    "precision, recall = evaluate(found_groups, true_matches)\n",
+    "print('\\n\\nWith blocking: ')\n",
+    "print(f'precision: {precision}, recall: {recall}')\n",
+    "\n",
+    "# matching without blocking\n",
+    "found_groups = naive_solve(clk_groups, threshold)\n",
+    "precision, recall = evaluate(found_groups, true_matches)\n",
+    "print('Without blocking: ')\n",
+    "print(f'precision: {precision}, recall: {recall}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assess Reduction Ratio\n",
```

Comment:
As mentioned above, I would call this `Assess blocking`.

Then one short paragraph that describes that the goal of blocking is to reduce the number of comparisons, we measure that with reduction ratio, whilst ensuring that entity pairs end up in the same block, -> pair completeness.